performance-testing/performance-testing-accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/performancetesting/ingest/TestAccumuloStoreRandomElementIngestTest.java

```diff
@@ -15,21 +15,30 @@
  */
 package uk.gov.gchq.gaffer.accumulostore.performancetesting.ingest;
 
+import org.apache.hadoop.conf.Configuration;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
+import uk.gov.gchq.gaffer.accumulostore.AccumuloProperties;
+import uk.gov.gchq.gaffer.accumulostore.AccumuloStore;
+import uk.gov.gchq.gaffer.accumulostore.MockAccumuloStore;
+import uk.gov.gchq.gaffer.graph.Graph;
 import uk.gov.gchq.gaffer.operation.OperationException;
+import uk.gov.gchq.gaffer.randomelementgeneration.Constants;
+import uk.gov.gchq.gaffer.randomelementgeneration.supplier.RmatElementSupplier;
 import uk.gov.gchq.gaffer.store.StoreException;
+import uk.gov.gchq.gaffer.store.schema.Schema;
 
 import java.io.File;
+import java.io.IOException;
 
 /**
  *
  */
 public class TestAccumuloStoreRandomElementIngestTest {
 
     @Rule
-    public TemporaryFolder tempFolder = new TemporaryFolder(new File("/tmp/"));
+    public TemporaryFolder tempFolder = new TemporaryFolder();
```

Comment:
Throughout the rest of Gaffer tests we tend to construct the temporary folders like this:

```java
new TemporaryFolder(CommonTestConstants.TMP_DIRECTORY);
```